src/cr/cube/cubepart.py

```diff
@@ -1145,6 +1162,22 @@ def table_weighted_bases(self):
         """2D np.float64 ndarray of table-proportion denominator for each cell."""
         return self._assembler.table_weighted_bases
 
+    @lazyproperty
+    def total_share_sum(self):
```

Comment:
Are we sure we want the `total_` instead of (maybe) `table_`?

Reply:
we do, we decide it with mike and joe, .... even the companion branch in zoom will have all the measure labels prefixed with `total` instead of `table` (i have to refactor them). So maybe in a nearly future I'm gonna refactor all the table_ measure names in cr.cube either